devel/sn-demangler/Portfile

```diff
@@ -6,13 +6,13 @@ PortGroup           java 1.0
 PortGroup           legacysupport 1.1
 PortGroup           compiler_blacklist_versions 1.0
 
-github.setup        indoorvivants sn-demangler 0.0.3 v
+github.setup        indoorvivants sn-demangler 0.0.7 v
 revision            1
```

Comment:
the `revision` field should be reset to `0` when updating the version

Reply:
@reneeotten thanks!